### PR TITLE
Add quick search to the home page

### PR DESCRIPTION
**Pull request by ersinciftci**

### CODE REVIEW

File: src/appBootstrapper.jsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ import getBrowserWindow from "shared/lib/getBrowserWindow";
 import {getConfigurationServiceApiUrl} from "shared/api/urls";
 import {AppStore} from "./AppStore";
 
-superagentCache(superagent);
+//superagentCache(superagent);
```
Comment (ersinciftci) [inline src/appBootstrapper.jsx]: Caching causes a bug in the search bar (if the load more option clicked for a term for the second time, the dropdown disappears), so I'm turning it off for now until I find a fix, so the heroku demo wouldn't be buggy.

**Comment by alisman** [comment]
replaced with https://github.com/cBioPortal/cbioportal-frontend/pull/1966

**Comment by jjgao** [comment]
@alisman is this replaced with #1966?

**Comment by inodb** [comment]
This is in release-2.2.0 branch now

**Pull request closed without merging**